multihost_test/bz_automation/test_automation.py

```diff
@@ -185,3 +185,55 @@ def test_pam_faillock_audit(self, multihost, create_localusers, bkp_pam_config):
         log_str = multihost.client[0].get_file_contents("/var/log/audit/audit.log").decode('utf-8')
         client.run_command("cp -vf /etc/security/faillock.conf_anuj /etc/security/faillock.conf")
         assert f'op=pam_faillock suid={uid}' in log_str
+
+    def test_6967(self, multihost, bkp_pam_config):
```

Comment:
Please use a more meaningful test name. The current one doesn't explain anything.